src/components/GroupAndSearchDropdown/index.js

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from 'react'
+import { Dropdown } from 'antd'
+import PropTypes from 'prop-types'
+import { Icon } from '../Icon'
+import { Text } from '../Typography'
+import { TextField } from '../Textfield/index'
+import './groupAndSearchDropdown.scss'
+
+export const GroupAndSearchDropdown = ({ options, onSearchChange, className, ...props }) => {
+  const [showDropdown, setShowDropdown] = useState(false)
+  const [value, setValue] = useState('')
+
+  const OptionsDropdown = () => {
+    return (
+      <div className="options-dropdown">
+        <div className="search-box">
+          <TextField type="search-box" placeholder="Search" onChange={onSearchChange} />
```

Comment:
How is the value cleared?
i.e what happens when you close and open the dropdown? Is the `searchValue` retained?